tests/model_tests.py

```diff
@@ -126,6 +126,28 @@ def test_select_star(self):
         )
         assert sql.startswith(expected)
 
+    def test_select_star_with_exotic_names(self):
+        main_db = get_example_database()
```

Comment:
@villebro I'm not overly familiar with the new example database but I gather the engine is related to the tox environment, i.e., `py36-mysql`, `py35-sqlite`, etc.

Reply:
It's my understanding that the example db falls back to `SQLALCHEMY_DATABASE_URI` if `SQLALCHEMY_EXAMPLES_URI` is undefined, i.e. it should test all engines that have a tox env.